### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by computeronix**
Modified result of bridge automation configuration file now called production

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Storj Bridge
-============
+============d
```
Comment (bookchin) [inline README.md]: Typo?

**Comment by super3** [comment]
@computeronix Ping.

**Comment by computeronix** [comment]
when discussing with Phillip the readme should discuss a production system setup     The automation script references production and also the manual setup instructions say to use production too

devel can be referenced as needed for development systems but we should be displaying a production system setup for users

Hope that makes sense :)

**Comment by bookchin** [comment]
@computeronix did you see my line comment? I think you've got a typo in the header.

**Comment by computeronix** [comment]
checking didnt see that

**Comment by computeronix** [comment]
corrected

**Pull request merged**